css-positioning/absolute.css

```diff
@@ -0,0 +1,33 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  padding-top: 50px;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 20px;
+  left: 240px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  padding-top: 140px;
+}
+
+.overlay {
```

Comment:
you need to update the code here so that the overlay displays on the page. It must cover the parent completely from top to bottom.